bin/generate_kong_config

```diff
@@ -22,7 +22,7 @@ echo "servers:
 announce "Generating Kong declarative configuration yaml"
 docker run --rm \
     -v ${PWD}:/opt/openapi \
-    node:latest /bin/bash -c "
+    node:15 /bin/bash -c "
```

Comment:
Should probably open an issue to get this working with the latest version of Node, not a huge deal but we probably don't want to keep using an old version forever.

Reply:
I will. I'm also going to file in the culprit repo. I think it's a few dependencies downstream:
`insomnia-inso -> node-libcurl -> node-pre-gyp`